### PR TITLE
Player logic

### PR DESCRIPTION
**Pull request by brehrey**
Finished suggest and accuse functions to game_board_controller.
Fixed logic to dole out remaining cards evenly to users on start of game.

Need to work on player turns and skipping disabled players if they made a bad accusation.

### CODE REVIEW

File: app/controllers/game_board_controller.rb [hunk 1]
```diff
@@ -128,25 +133,30 @@ def make_suggestion
...
+        if r.name == room
+          location_id = r.id
+          break
         end
-
-        if p.get_cards[:weapon].name == weapon
-          results.push({:player => p, :card => weapon})
+      end
+      
+      $game.get_players.collect { |k, v| v }.each do |p|
```
Comment (billyyarosh) [inline app/controllers/game_board_controller.rb:146]: I think you can do:

``` ruby
$game.get_players.each do |index, player| 

end

```

Comment (brehrey) [inline app/controllers/game_board_controller.rb:146]: Ok, I just grabbed the method that was used in the other methods. That way
looks a lot cleaner though.

File: lib/game.rb [hunk 2]
```diff
@@ -58,14 +59,19 @@ def player_in_turn
...
-
-    # TODO figure out how to assign random cards.
-    @players.each do |player|
-      card = available_cards.random_card
-      break unless card
-
-      player.cards = player.cards + card
+    
+    #Distribute remaining cards
+    while @available_cards.card_list.length > 0 do
```
Comment (billyyarosh) [inline lib/game.rb:64]: :+1:

**Comment by billyyarosh** [comment]
good job +1

**Pull request merged**